<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	color: rgb(0, 123, 255);	
	background-color: rgb(0, 0, 0);

}

QWidget:item:selected
{
	color: rgb(239, 151, 0);
    border: 1px solid rgb(239, 151, 0);
	/*color: rgb(255, 255, 255);	
	background-color: rgb(0, 123, 255);	*/
}

QTabWidget::pane {
    border: 1px solid  rgb(97, 97, 97);
    top: 1px;
}

QTabBar::tab {
	background-color: rgb(97, 97, 97);
	color: rgb(255, 255, 255);
    border-width: 1px;
	border-color: rgb(255, 255, 255);
	border-style: solid;
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
	padding: 5px;
	padding-left: 63px;
	padding-right:63px;
}

QTabBar::tab:selected {
	color: rgb(255, 255, 255);	
	background-color: rgb(0, 123, 255);
}

.QMessageBox {font-size: 16pt}

.QSlider {
    min-height: 30px;
    max-height: 50px;
}

.QSlider::groove:horizontal {
    border: 1px solid;
    height: 5px;
    background: #393939;
    margin: 0 12px;
}

.QSlider::handle:horizontal {
    background: rgb(0, 123, 255);
    width: 30px;
    height: 30px;
    margin: -24px -12px;
}

QPushButton {
	background-color: rgb(97, 97, 97);
	color: rgb(255, 255, 255);
    border-width: 1px;
	border-color: rgb(97, 97, 97);
	border-style: solid;
	border-radius: 7;
	padding: 3px;
	padding-left: 5px;
	padding-right: 5px;
}

QPushButton:checked
{
	color: rgb(255, 255, 255);	
	background-color: rgb(0, 123, 255);
	border-color: rgb(0, 123, 255);
}

QPushButton:pressed
{
	color: rgb(255, 255, 255);	
	background-color: rgb(0, 123, 255);
	border-color: rgb(0, 123, 255);
}

QProgressBar
{
	background-color: rgb(166, 166, 166);
	color: rgb(0, 0, 0);
    border: 2px rgb(166, 166, 166);
    border-radius: 5px;
}

QProgressBar::chunk {
	border: 2px rgb(166, 166, 166);
    border-top-left-radius: 5px;
    border-bottom-left-radius: 5px;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
	background-color: rgb(0, 123, 255);
   /* width: 40px;
    margin: 0.5px;*/
}

QComboBox {
	color: rgb(0, 123, 255);	
    border: 1px solid  rgb(97, 97, 97);
    border-top-left-radius: 7px;
    border-bottom-left-radius: 7px;
    padding: 1px 18px 1px 5px;
    min-width: 6em;
}

QComboBox:on { /* shift the text when the popup opens */
    border: 1px solid  rgb(97, 97, 97);
    border-top-left-radius: 7px;
    border-bottom-left-radius: 0px;
	color: rgb(239, 151, 0);
}

/* QComboBox:hover
{
	color: rgb(239, 151, 0);
}*/

QComboBox QAbstractItemView {
    border: 1px solid  rgb(97, 97, 97);
}

/*QComboBox:item:checked {
	color: rgb(239, 151, 0);
    border: 1px solid rgb(239, 151, 0);
}

QComboBox:item:selected {
	color: rgb(239, 151, 0);
    border: 1px solid rgb(239, 151, 0);
} */

QLineEdit
{
    padding: 1px;
    border-style: solid;
    border: 1px solid rgb(97, 97, 97);
    border-radius: 5;
}

#LE_curr_name{
	color: rgb(239, 151, 0);
}

#LE_new_name{
	color: rgb(239, 151, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>26</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Aktueller Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>26</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Neuer Name:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="LE_curr_name">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="maxLength">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClickableLineEdit" name="LE_new_name">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="maxLength">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Auf Karte schreiben</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>clickablelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
